Jenkinsfile

```diff
@@ -101,7 +92,7 @@ pipeline {
                   unstash 'autopas_exec'
                   dir ("build"){
                     sh """
-                      ./src/MarDyn ../examples/Argon/200K_18mol_l/config_autopas_aos.xml --steps=20 | tee autopas_run_log.txt
+                      ./src/MarDyn ../examples/Argon/200K_18mol_l/config_autopas_aos.xml --steps=20 --legacy-cell-processor | tee autopas_run_log.txt
```

Comment:
this change should not be needed

Reply:
I just figured because it said testing for AOS and SOA... So this is only supposed to use AOS in the autopas components?

Reply:
it doesn't have any influence to autopas. so not specifying it is the better approach.